src/pspm_pupil_correct_eyelink.m

```diff
@@ -121,9 +110,10 @@
 if options.invalid
   return
 end
+
 if strcmp(options.mode, 'manual')
   for field = all_fieldnames
-    if ~isfield(options, field{1})
+    if ~isfield(options, field{1}) || options.(field{1}) == 0
```

Comment:
I tested this line of code in MATLAB and I am not sure it works..
<img width="217" alt="image" src="https://github.com/bachlab/PsPM/assets/3895146/46a5aafa-992a-440d-8410-5477de2ee474">

Reply:
I think there is a `~` missing in your test. You want to check the following code instead:
```
a = struct();
~isfield(a, 'c') || a.c == 0
```

Reply:
> ~isfield(a, 'c') || a.c == 0

Thanks, I understand now. The two clause `~isfield(a, 'c')` and `a.c == 0` are not at the same level. `a.c==0` will be checked only if `isfield(a,'c')` is true or `~isfield(a,'c')` is false.